tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -217,78 +217,111 @@ def sumIsLargerFunc(value, row_schema):
     return sumIsLargerFunc
 
 
+def recordHasLength(length):
```

Comment:
i moved the following functions for clarity, but haven't modified them. they are used as preparsing validators, so we pass `0` in for the field number, and `-1` for item number. this results in each error being prefixed with `{record_type} Item -1 record type: {error message}`, where `record type` is not the field related to the error. I left the messages un-touched for clarity but open to suggestions on how to address this